index.html

```diff
@@ -117,9 +123,9 @@ <h2 class="work-title">Tonic</h2>
                     <button type="button" class="see-more">See Project</button>
```

Comment:
> Your project looks great! you have done an amazing job, I even thought that it was just a screenshot from Figma 😄. 
- [ ] I only want to recommend to improve buttons on your page to make it even better. Did you know that your provided with a set of button styles for `hover` `pressed` states as well? If not then check screenshot below:
<img width="634" alt="Screenshot 2022-03-24 at 13 54 14" src="https://user-images.githubusercontent.com/77038610/159890335-d0cc4f08-006d-4175-a552-ce980288973b.png">

- [ ] Please make sure add different styles on hover and pressed states of the button. It would also be nice to add `cursor: pointer`, it will improve user experience  👍🏻